packages/clarity-js/src/interaction/scroll.ts

```diff
@@ -45,13 +48,17 @@ function recompute(event: UIEvent = null): void {
     const yOffset = width > height ? height * 0.15 : height * 0.2;
     const startYPosition = yOffset;
     const endYPosition = height - yOffset;
-    const top = getPositionHash(xPosition, startYPosition);
-    const bottom = getPositionHash(xPosition, endYPosition);
+    const top = getPositionNode(xPosition, startYPosition);
+    const bottom = getPositionNode(xPosition, endYPosition);
 
     let current: ScrollState = { time: time(event), event: Event.Scroll, data: {target: element, x, y, top, bottom} };
 
     // We don't send any scroll events if this is the first event and the current position is top (0,0)
-    if ((event === null && x === 0 && y === 0) || (x === null || y === null)) { return; }
+    if ((event === null && x === 0 && y === 0) || (x === null || y === null)) {
+        initTopNode = top;
+        initBottomNode = bottom;
```

Comment:
I thought part of the issue here was that the scroll event didn't fire until the scroll had actually completed. Don't you need to grab the initial top/bottom nodes before any scroll events?

Reply:
this gets called when the scroll module starts, before any scroll event happens